modules/uid2IdSystem.md

```diff
@@ -1,264 +1,84 @@
-| params.phone | Optional, Client-side token generation | String | The user's phone number. Provide this parameter if using phone as the DII. | `"+15555555555"` |
-| params.phoneHash | Optional, Client-side token generation | String | A hashed, normalized representation of the user's phone number. Provide this parameter if using phoneHash as the DII. | `"tMmiiTI7IaAcPpQPFQ65uMVCWH8av9jw4cwf/F5HVRQ="` |
-
-# Normalization and Encoding
-
-This section provides information about normalizing and encoding [directly Identifying information (DII)](https://unifiedid.com/docs/ref-info/glossary-uid#gl-dii). It's important that, in working with UID2, normalizing and encoding are performed correctly.
+| name | Required | String | ID value for the UID2 module. Must be `"uid2"`. | `"uid2"` |
+| params.uid2ApiBase | Optional | String | Overrides the default UID2 API endpoint. | `"https://prod.uidapi.com"` _(default)_|
+| params.storage | Optional | String | Specify whether to use `cookie` or `localStorage` for module-internal storage. It is recommended to not provide this and allow the module to use the default. | `localStorage` _(default)_ |
 
-## Introduction
-When you're taking user information such as an email address, and following the steps to create a raw UID2 and/or a UID2 advertising token, it's very important that you follow all the required steps. Whether you normalize the information or not, whether you hash it or not, follow the steps exactly. By doing so, you can ensure that the UID2 value you create can be securely and anonymously matched up with other instances of online behavior by the same user.
+### Client-side Integration
 
->Note: Raw UID2s, and their associated UID2 tokens, are case sensitive. When working with UID2, it's important to pass all IDs and tokens without changing the case. Mismatched IDs can cause ID parsing or token decryption errors.
+The following parameters apply to the UID2 User ID Module if you are following the client-side integration guide. Exactly one of params.email, params.emailHash, params.phone, and params.phoneHash must be provided.
 
-## Types of Directly Identifying Information
-UID2 supports the following types of directly identifying information (DII):
-- Email address
-- Phone number
-
-## Email Address Normalization
-
-If you send unhashed email addresses to the UID2 Operator Service, the service normalizes the email addresses and then hashes them. If you want to hash the email addresses yourself before sending them, you must normalize them before you hash them.
+| Param under userSync.userIds[] | Scope | Type | Description | Example |
+| --- | --- | --- | --- | --- |
+| params.serverPublicKey | Required for client-side integration | String | A public key for encrypting the DII payload for the Operator's CSTG endpoint. | - |
```

Comment:
Not sure what we mean by CSTG endpoint here.

Reply:
Good catch, thanks!